configuration/reports/reportdescriptors/dataexports/sql/j9CaseRegistration.sql

```diff
@@ -73,47 +96,82 @@ set t.department = cna.department,
     t.commune = cna.commune,
     t.section_communal = cna.section_communal,
     t.locality = cna.locality,
-    t.street_landmark = cna.street_landmark
-;
+    t.street_landmark = cna.street_landmark;
 
+   -- identifiers
 update temp_J9_patients t
-set t.ZL_identifier =  zlEMR(t.patient_id)
-;
+set t.ZL_identifier =  zlEMR(t.patient_id);
 
 update temp_J9_patients t
-set t.Dossier_identifier =  dosId(t.patient_id)
-;
+set t.Dossier_identifier =  dosId(t.patient_id);
 
 -- patient names
 update temp_J9_patients t
     left outer join current_name_address cna on cna.person_id = t.patient_id
 set t.given_name = cna.given_name,
-    t.family_name = cna.family_name
```

Comment:
Same here regarding use of current_name_address.

Reply:
same as above